torch/nn/functional.pyi.in

```diff
@@ -47,36 +52,66 @@ def _canonical_mask(
     target_type: _dtype,
     check_other: bool = True,
 ) -> Tensor | None: ...
+
+__all__ += ["_canonical_mask"]
```

Comment:
nit: I wonder if it would be better to use list.append() for all these to avoid creating an extra rhs list
```suggestion
__all__.append("_canonical_mask")
```

Reply:
It is suggested to use `+=` instead of `.append()` and `.extend()` on `__all__`. See https://docs.astral.sh/ruff/rules/unsupported-method-call-on-all.